src/components/header/AppHeader.vue

```diff
@@ -88,15 +90,6 @@ const handleSelect = (key: string | number) => {
             </n-button>
           </RouterLink>
 
-          <RouterLink to="/auth/signup" v-if="!authStore.isAuthenticated">
```

Comment:
Why was this button removed?

Reply:
i don't think i removed, either accidentally, or i merged my branch with staging before push, it came from there